packages/synapse-interface/utils/calculateEstimatedTransactionTime.tsx

```diff
@@ -6,8 +6,12 @@ import { ARBITRUM, ETH } from '@/constants/chains/master'
 import { USDC } from '@/constants/tokens/bridgeable'
 import { CHAINS_BY_ID } from '@/constants/chains'
 
-// Utility function to fetch estimated transaction time
-// Returned as a number, in seconds
+/**
+ * Utility function to fetch estimated transaction time
+ * @param originChainId
+ * @param originTokenAddress
+ * @returns estimated bridge duration, in seconds
+ */
 export const calculateEstimatedTransactionTime = ({
```

Comment:
Created a new util function to fetch estimated bridge time using the SDK updates @ChiTimesChi made a few weeks back: 

https://github.com/synapsecns/sanguine/blob/feat/bridge-receipt/packages/synapse-interface/utils/getEstimatedBridgeTime.tsx#L18

We should be able to just replace any instances of where `calculateEstimatedTransactionTime()` occurs with `getEstimatedBridgeTime()`